nautobot/ui/lib/nautobot-craco-alias-plugin.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * This is a wrapper around react-app-alias and react-app-alias-ex that addresses
+ * a bug in react-app-alias-ex where it does not properly resolve the path to
+ * node_modules when running Jest tests.
+ *
+ * See: https://github.com/oklas/react-app-alias/issues/95
```

Comment:
@bryanculver your latest comments on this issue make it sound like #3618 may have removed the need for this wrapper/workaround - still needed?

Reply:
Yes, @gsnider2195 still ran into an issue with this when getting the code output from Jest. So it worked for the tests but couldn't decompile. FWIW, the reference issue upstream has been closed with the patch merged in.